src/commands/app/dev/index.js

```diff
@@ -20,23 +20,31 @@ const open = require('open')
 const { Flags, ux } = require('@oclif/core')
 const coreConfig = require('@adobe/aio-lib-core-config')
 
-const BaseCommand = require('../../BaseCommand')
-const runDev = require('../../lib/run-dev')
-const SERVER_DEFAULT_PORT = 9080
-// const { runInProcess } = require('../../lib/app-helper')
+const BaseCommand = require('../../../BaseCommand')
+const runDev = require('../../../lib/run-dev')
 
+const SERVER_DEFAULT_PORT = 9080
 const DEV_KEYS_DIR = 'dist/dev-keys/'
 const PRIVATE_KEY_PATH = 'dist/dev-keys/private.key'
 const PUB_CERT_PATH = 'dist/dev-keys/cert-pub.crt'
 const CONFIG_KEY = 'aio-dev.dev-keys'
 
 class Dev extends BaseCommand {
   async run () {
-    // cli input
     const { flags } = await this.parse(Dev)
 
     const spinner = ora()
 
+    // run `aio app dev init`
```

Comment:
Adding an init command makes sense, but I don't understand why we need to call it whenever we run `app dev`.  The checks don't make a lot of sense to me either, this command absolutely knows that app:dev:init is installed, because it is the same command ... so why the extra checks?

imho, `app dev init` should work just like `git init` .. ie, you run it once, and it is only there to add the vscode config to the project, and you cannot init more than once in a project ...

Reply:
this is probably a remnant of when the .vscode/launch.json actually contained the ports needed for Chrome debugging (to launch the browser) - since the port is parsed from the log we don't need it anymore, so we can remove this init.

> I am not sure the bundlePort vs serverPort logic belongs in this pr. This seems related to frontend hot reloading.

Related, yes. But it's not quite a hot reload, but more of a "web source changed? re-run the bundler for the web-src". The dev would have to manual reload. Initially I wanted to put both PRs into this one, I guess I should have...

Reply:
removed in https://github.com/adobe/aio-cli-plugin-app-dev/pull/14/commits/f0f3fdca3a465c3ddca3c3f9d634dcfd490e438a 
I'll leave the bundle watcher in since we will be pulling in #15 next anyway